skbio/sequence/__init__.py

```diff
@@ -24,7 +24,7 @@
 
    Sequence
    IUPACSequence
-   NucleotideSequence
+   NucleotideMixin
```

Comment:
Is there a reason to export IUPACSequence and not NucleotideMixin?
On Jun 28, 2015 11:55 AM, "Jai Ram Rideout" <notifications@github.com>
wrote:

> In skbio/sequence/__init__.py
> <https://github.com/biocore/scikit-bio/pull/968#discussion_r33425711>:
>
> > @@ -24,7 +24,7 @@
> >
> >     Sequence
> >     IUPACSequence
> > -   NucleotideSequence
> > +   NucleotideMixin
>
> don't export
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/968/files#r33425711>.
>